cmd/deployment.go

```diff
@@ -148,22 +161,37 @@ func newDeploymentListCmd(client *houston.Client, out io.Writer) *cobra.Command
 }
 
 func newDeploymentUpdateCmd(client *houston.Client, out io.Writer) *cobra.Command {
+	example := ` 
+# update lables and description for given deployment
+$ astro deployment update UUID label=Production-Airflow description=example version=v1.0.0`
+	updateExampleDagDeployment := `
+
+# update dag deployment strategy
+$ astro deployment update UUID label=Production-Airflow --dag-deployment-type=volume --nfs-location=test:/test`
```

Comment:
Are we not prepending the protocol name "nfs://" like we did in the UI?
![Screen Shot 2021-04-26 at 2 54 29 PM](https://user-images.githubusercontent.com/60400338/116155720-5752ff00-a69f-11eb-8a09-c96e9b62a20a.png)

Reply:
@vishwas-astro what do you mean?

Reply:
Is the nfs-location field just free formed text? I thought we were always assuming it is a string that starts with "nfs://". That is what I had assumed from the UI.

Reply:
same string as in UI. we don't have ability in cli to add labels/or prefix `nfs://` on command line arguments.
If you revisit help text you can see we have example:
```
-n, --nfs-location string          NFS Volume Mount: <IP>:/<path>
```

may be we can extend it with something:

```
-n, --nfs-location string          NFS Volume Mount: <IP>:/<path> without protocol `nfs://`
```
```
-n, --nfs-location string          NFS Volume Mount: <IP>:/<path>. Please avoid `nfs://` protocol.
```

cc @paolaperaza @ryanahamilton

Reply:
also we have example:
```
Examples:

# update labels and description for given deployment
$ astro deployment update UUID label=Production-Airflow description=example version=v1.0.0

# update dag deployment strategy
$ astro deployment update UUID label=Production-Airflow --dag-deployment-type=volume --nfs-location=test:/test
```

Reply:
I like something like
-n, --nfs-location string          NFS Volume Mount: <IP>:/<path>. `nfs://` protocol is assumed, don't include it.

@jwitz or @paolaperaza, can you please bless?

Reply:
Attempting to focus on what the CLI does, rather than what the user should do: `NFS Volume Mount, specified as: <IP>:/<path>. Input is automatically prepended with nfs://`

Reply:
I wouldn't mind being explicit here, as otherwise we're forcing the user to logically jump from "This is automatically prepended" to in practice understanding that this means "Ok so I shouldn't include it."

I'd suggest: `NFS Volume Mount, specified as: <IP>:/<path>. Do not include "nfs://" - input is automatically prepended.`

Reply:
done:
```
Examples:

# Create new deployment with Celery executor (default: celery without params).
  $ astro deployment create new-deployment-name --executor=celery

# Create new deployment with Local executor.
  $ astro deployment create new-deployment-name-local --executor=local

# Create new deployment with Kubernetes executor.
  $ astro deployment create new-deployment-name-k8s --executor=k8s

# Create new deployment with Kubernetes executor.
  $ astro deployment create my-new-deployment --executor=k8s --airflow-version=1.10.10

# Create new deployment with Kubernetes executor and dag deployment type volume and nfs location.
  $ astro deployment create my-new-deployment --executor=k8s --airflow-version=1.10.10 --dag-deployment-type=volume --nfs-location=test:/test


Flags:
  -a, --airflow-version string       Add desired airflow version parameter: e.g: 1.10.5 or 1.10.7
  -c, --cloud-role string            Set cloud role to annotate service accounts in deployment
  -t, --dag-deployment-type string   DAG Deployment mechanism: image, volume (default "image")
  -e, --executor string              Add executor parameter: local, celery, or kubernetes
  -h, --help                         help for create
  -n, --nfs-location string          NFS Volume Mount, specified as: <IP>:/<path>. Input is automatically prepended with 'nfs://'
  -r, --release-name string          Set custom release-name if possible

Global Flags:
      --skip-version-check    skip version compatibility check
      --workspace-id string   workspace assigned to deployment
```

Reply:
same for update:
```
pdate airflow deployments

Usage:
  astro deployment update [flags]

Aliases:
  update, up

Examples:

# update labels and description for given deployment
$ astro deployment update UUID label=Production-Airflow description=example version=v1.0.0

# update dag deployment strategy
$ astro deployment update UUID label=Production-Airflow --dag-deployment-type=volume --nfs-location=test:/test

Flags:
  -c, --cloud-role string            Set cloud role to annotate service accounts in deployment
  -t, --dag-deployment-type string   DAG Deployment mechanism: image, volume (default "image")
  -h, --help                         help for update
  -n, --nfs-location string          NFS Volume Mount, specified as: <IP>:/<path>. Input is automatically prepended with 'nfs://'

Global Flags:
```

Reply:
@paolaperaza given that space is a premium on help text, I think it's okay to leave it as an assumption. The logical jump we're expecting users to do is manageable, imo. Grammatically as well, "Input is automatically prepended" doesn't seem right to me. Automatically prepended with what? It becomes another logical jump we're expecting users to understand.

Reply:
I like being explicit. @andriisoldatenko , please make the latest suggested change from @paolaperaza

Reply:
done @vishwas-astro